File: 351003/Zhigimont/distcomp/src/main/java/com/bsuir/distcomp/dto/TopicResponseTo.java
package com.bsuir.distcomp.dto;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class TopicResponseTo {

    private Long id;
    private Long writerId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;

}

File: 351001/Strizhak/rest_jpa/rest_jpa/Dtos/TopicResponseTo.cs
namespace rest_api.Dtos
{
    public class TopicResponseTo
    {
        public long Id { get; set; }
        public long UserId { get; set; }
        public string Title { get; set; } = null!;
        public string Content { get; set; } = null!;
        public DateTime Created { get; set; }
        public DateTime Modified { get; set; }
    }
}
